### ISSUE TITLE
get_member_entitlement_management_client is not part of the release package in the client_factory.py

### ISSUE THREAD

**Issue by mahhanifi** [microsoft/azure-devops-python-api#273]
Hi,
Trying to use the `get_member_entitlement_management_client()` but it is not available in the `released\client_factory.py' is there a specific reason or it's just left out?

**Comment by tedchamb** [comment]
While most 5.1 APIs are released, there are some APIs that were introduced at version 5.1 or an earlier version, but were not ready to be released at the 5.1 release cutoff. These are still marked as -preview, and will most likely be released in 6.0.  Each team is in control of when their APIs are considered released.  Once that is done the python SDK, will automatically pick that up when we regenerate the clients.   Its possible that it is just an oversight that the Member Entitlement Management area's team has not released their APIs.  I will followup with them.  If so, the next opportunity for them to correct it will be for the 6.0 version release.
